actions/bumper/entrypoint.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/sh
+
+# based off https://github.com/anothrNick/github-tag-action
+
+set -eu
+
+# input configuration and defaults
+RELEASE_BRANCH_LIST=${INPUT_RELEASE_BRANCH_LIST:-develop,master}
+DEFAULT_SEMVER_BUMP=${INPUT_SEMVER_BUMP:-minor}
+VERSION_FILE_PATH=${INPUT_VERSION_FILE_PATH:-}
```

Comment:
Whats the point of `:-` here if there's no default?

Reply:
It makes the variable an empty string as opposed to undefined, which would otherwise trigger the `set -eu` and error out.